mt-support/tribe-helpdesk
# Fix/125988 header navbar fix

Updates for 125988: 
- Updated header to cap at site width instead of running full width
- Updated body padding to accommodate for the space when nav goes to fixed > fixes "jumpiness"

Updates for 125989:
- Update to use the Bootstrap "Affix" plugin (already included in LiveAgent JS bundle)
- Update CSS accordingly

**QA Notes**
You can open up the "kb.html" file which should mimic an actual KB article page. Things to note:

1. When scrolling down the page, the navbar should fix to the top but no longer expand to the width of the viewport. There should also no longer be much of a "jump" when scrolling down or up
2. When scrolling the main content area, the sidebar should "stick" to the top of the viewport. Transition should be smooth and not "jumpy"
3. Reduce the viewport to below 768px. Verify the mobile menu displays. Increase viewport width above 768px and ensure the nav menu items are in line with the logo and not "stacked"
